composer.json

```diff
@@ -53,7 +53,8 @@
   "require": {
     "php": ">=5.6.0",
     "composer/installers": "~1.0",
-    "oomphinc/composer-installers-extender": "^1.1"
+    "oomphinc/composer-installers-extender": "^1.1",
+    "ext-simplexml": "*"
```

Comment:
PHPStorm code inspections is complaining about this being missing in composer.json.
https://getcomposer.org/doc/01-basic-usage.md#platform-packages

It will help ensure the developers custom setup can run the code so it's worth putting in.